src/build_settings.cpp

```diff
@@ -510,15 +510,15 @@ gb_global TargetMetrics target_darwin_amd64 = {
 	TargetOs_darwin,
 	TargetArch_amd64,
 	8, 8, 8, 16,
-	str_lit("x86_64-apple-darwin"),
+	str_lit("x86_64-apple-macosx"), // NOTE: Changes during initialization based on build flags.
```

Comment:
We'll need to change this string them when we support `iOS`

Reply:
It is already changed to "arm64-apple-ios" or "x86_64-apple-ios" when you use `-subtarget:ios`